src/ILCompiler/src/Program.cs

```diff
@@ -168,6 +173,36 @@ private int Run(string[] args)
                 throw new CommandLineException("Output filename must be specified (/out <file>)");
 
             //
+            // Set target Architecture and OS
+            //
+            if (_targetArchitectureStr != null)
+            {
+                if (_targetArchitectureStr.Equals("x86"))
```

Comment:
It maybe nice to do the comparisons case-insensitive for convenience. `_targetArchitectureStr.Equals("x86", StringComparison.OrdinalIgnoreCase)`